### PR TITLE
Add 0.4.x release announcements

### PR DESCRIPTION
**Pull request by axic**
Part of #31.

### CODE REVIEW

File: _posts/2016-09-17-solidity-0.4.2-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.4.2 Release Announcement'
+date: '2016-09-17'
+author: Solidity Team
+category: Releases
+---
+
```
Comment (franzihei) [inline _posts/2016-09-17-solidity-0.4.2-release-announcement.md:9]: Maybe we can add an introductory one-liner here "This is a bugfix release fixing..."?

File: _posts/2017-01-13-solidity-0.4.8-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.4.8 Release Announcement'
+date: '2017-01-13'
+author: Solidity Team
+category: Releases
+---
+
```
Comment (franzihei) [inline _posts/2017-01-13-solidity-0.4.8-release-announcement.md:9]: Introduction sentence is missing here, too. I'm not sure if it's worth the effort adding one, I'm just pointing it out. :)

File: _posts/2016-12-15-solidity-0.4.7-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.4.7 Release Announcement'
+date: '2016-12-15'
+author: Solidity Team
+category: Releases
+---
+
```
Comment (franzihei) [inline _posts/2016-12-15-solidity-0.4.7-release-announcement.md:9]: No introduction one-liner?

File: _posts/2016-11-22-solidity-0.4.6-release-announcement.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.4.6 Release Announcement'
+date: '2016-11-22'
+author: Solidity Team
+category: Releases
+---
+
```
Comment (franzihei) [inline _posts/2016-11-22-solidity-0.4.6-release-announcement.md:9]: Introduction one-liner missing.

**Comment by axic** [comment]
I think not having the intro sequence is fine, this is just historical, and mostly for google search results :wink:

**Pull request merged**